Revit_Core_Engine/Convert/Physical/ToRevit/FamilyInstance.cs

```diff
@@ -84,17 +84,15 @@ public static FamilyInstance ToRevitFamilyInstance(this Column framingElement, D
                 return null;
             }
 
-            if (((BH.oM.Geometry.Line)framingElement.Location).Start.Z >= ((BH.oM.Geometry.Line)framingElement.Location).End.Z)
-            {
-                BH.Engine.Base.Compute.RecordError(string.Format("Start point of Revit columns need to have lower elevation than the end point. Have a look at flipping your location curves. BHoM_Guid: {0}", framingElement.BHoM_Guid));
+            BH.oM.Geometry.Line locationLine = framingElement.ColumnLine();
```

Comment:
There is no need for the two checks above (`if (framingElement.Location == null)` and `if (framingElement.Location as BH.oM.Geometry.Line == null)` as you are checking this inside `ColumnLine` method.

Reply:
Thanks for the great tip! I mistakenly thought these methods did the same thing :scream: